pkg/crc/machine/start.go

```diff
@@ -290,10 +290,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	}
 
 	// Start network time synchronization if `CRC_DEBUG_ENABLE_STOP_NTP` is not set
-	if stopNtp, _ := strconv.ParseBool(os.Getenv("CRC_DEBUG_ENABLE_STOP_NTP")); !stopNtp {
-		logging.Info("Starting network time synchronization in CodeReady Containers VM")
-		if _, _, err := sshRunner.Run("sudo timedatectl set-ntp on"); err != nil {
-			return nil, errors.Wrap(err, "Failed to start network time synchronization")
+	if stopNtp, _ := strconv.ParseBool(os.Getenv("CRC_DEBUG_ENABLE_STOP_NTP")); stopNtp {
+		logging.Info("Stopping network time synchronization in CodeReady Containers VM")
+		if _, _, err := sshRunner.Run("sudo timedatectl set-ntp off"); err != nil {
+			return nil, errors.Wrap(err, "Failed to stop network time synchronization")
```

Comment:
Not sure this is going to be enough, `test/e2e/features/cert_rotation.feature` sets the host time 3 months ahead, and knows for sure the VM will use the host time rather than getting it from NTP since NTP was disabled at boot time. Now this has changed, and this looks like it could be racy, the test sets the host clock, the VM starts and syncs to the correct time using NTP, and then we stop NTP. We'd need an additional `ssh` command to set the VM time to be the same as the host time.

Reply:
@cfergeau yes, we also need to set the VM time ahead of 3 month after stopping the time sync so that when kubelet starts it sees that time and start cert rotation.